src/locales/en.ts

```diff
@@ -396,6 +396,11 @@ export default {
       alertMessage:
         'Request to servers have timed out and we cannot add your Hotspot at this time.\n\nPlease contact support@helium.com and note MAC address %{mac}.',
     },
+    skip_location: {
```

Comment:
We should add these to the other language files as well. cc: @cokes518

Reply:
they'll auto populate when we send the en.json out for translation.